lintconfig.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
if this file is generated, does it need to be checked in ?

Reply:
makes sense to me, not much reasons (I had this since it already has `generated_files` file).